### DISCUSSION TITLE
Search configured registries?

### DISCUSSION THREAD

**Discussion by luckman212** [podman-desktop/podman-desktop#9266] [Q&A]
Sorry for this dumb question...

How can I search e.g. Docker Hub registry from Podman Desktop? macOS 15.0.1 + Podman Desktop 1.12.0

I configured Docker Hub & GitHub (using personal access token)... 
<img width="1042" alt="image" src="https://github.com/user-attachments/assets/6382acdf-9c13-448c-a9eb-1c64f076145c">

I can search via CLI...
```
$ podman search --filter "stars=10" --format "table {{.Index}} {{.Name}} {{.Stars}}" ollama
INDEX       NAME                       STARS
docker.io   docker.io/ollama/ollama    768
docker.io   docker.io/ollama/quantize  16
docker.io   docker.io/litellm/ollama   14
```

But I can't search from inside Podman Desktop...
<img width="1050" alt="image" src="https://github.com/user-attachments/assets/8730c27e-7038-45a2-8c3b-c55dcd481561">

**Comment by benoitf** [comment]
Hello this feature is in upcoming 1.13

https://github.com/containers/podman-desktop/pull/8786

**Reply by luckman212** [reply]
Thanks 🙏 I did not know about that.

**Comment by benoitf** [comment]
you can find pre releases from there
https://github.com/containers/podman-desktop-prereleases/releases
